packages/theme/stories/characteristics/characteristics.stories.mdx

```diff
@@ -57,7 +57,7 @@ theme, --salt-actionable-primary-background resolves to `var(--salt-action-prima
 
 In the following example, we will show how to customize the Card component through characteristics. Currently, Card only has one variant, using tokens such as `---salt-container-primary-background` all of its styling purposes. Here's how it looks by default, shown on a purple Panel for demo purposes:
 
-<Panel style={{ background: "var(--salt-color-purple-10)" }}>
+<Panel style={{ "--saltPanel-background": "var(--salt-color-purple-10)" }}>
```

Comment:
We currently don't want to recommend this override behaviour. This change should be discussed with design